core/lib/spree/core/product_duplicator.rb

```diff
@@ -72,7 +72,7 @@ def reset_properties
     end
 
     def sku_generator(sku)
-      "COPY OF #{Variant.where('sku like ?', "%#{sku}").order(:created_at).last.sku}"
+      "COPY OF #{Variant.with_deleted.where('sku like ?', "%#{sku}").order(:created_at).last.sku}"
```

Comment:
@damianlegawiec Using `.unscoped` in place of `.with_deleted` is a better idea. 
Changes done. Please review :)